packages/ui/src/components/Checkbox/style.less

```diff
@@ -6,110 +6,54 @@
 @checkbox-border-width: @condo-global-border-width-default;
 @checkbox-border-radius: @condo-global-border-radius-small;
 @checkbox-size: (@condo-global-spacing-24 - (@checkbox-border-width * 2));
-@checkbox-inner-border-radius: @checkbox-border-radius - @checkbox-border-width;
 
 @condo-checkbox-checkmark-top: ((@checkbox-size / 2));
 @condo-checkbox-checkmark-left: ((@checkbox-size / 4) - 0.5);
 @condo-checkbox-checkmark-width: (@checkbox-size / 4) - 1;
 @condo-checkbox-checkmark-height: (@checkbox-size / 2) - 1.5;
 
+@condo-checkbox-focus-outline-width: @condo-global-border-width-default * 2;
+@condo-checkbox-focus-layer-margin: @condo-checkbox-focus-outline-width + @checkbox-border-width + 2px;
 @condo-checkbox-hidden-area: @condo-global-spacing-8;
 
 .condo-checkbox-wrapper {
   align-items: center;
 
+  * {
+    transition: all @condo-global-transition-duration-default;
+  }
+
   .condo-checkbox {
     top: 0;
+    border-color: @condo-global-color-gray-5;
 
     &-input {
       margin: 0;
     }
 
     &-inner {
       box-sizing: content-box;
-      border-color: @condo-global-color-gray-5;
-
-      &::after {
-        transition: none;
-      }
-    }
-
-    &-indeterminate > .condo-checkbox-inner {
-      &::after {
-        width: @condo-global-spacing-12;
-        height: @condo-global-spacing-4;
-        background: @condo-global-color-brand-gradient-5 border-box;
-        border-radius: @condo-global-border-width-default;
-      }
-
-      &::before {
-        .condo-checkbox-inner-layer(@condo-global-color-brand-gradient-1, @checkbox-inner-border-radius);
-      }
-    }
-
-    &-checked::after {
-      border: none;
-    }
-
-    &-checked > .condo-checkbox-inner {
-      box-sizing: content-box;
-      background: @condo-global-color-gray-5 border-box;
-      border-color: transparent;
+      border-color: @condo-global-color-gray-5 !important;
```

Comment:
transparent was working without important flag, why gray doesn't?)

Reply:
Previously, there was also important flag, but only in a different place

Reply:
fixed